doc/database/database_guidelines.md

```diff
@@ -0,0 +1,33 @@
+# Database Guidelines
+
+This document outlines the conventions and guidelines for designing and maintaining a well-structured and organized database.
+Please follow these guidelines to ensure consistency and ease of use across the database.
+## 1. Table Names
```

Comment:
If not yet required, we should probably ask prettier to add newlines before/after titles.

```suggestion
Please follow these guidelines to ensure consistency and ease of use across the database.

## 1. Table Names
```